packages/gatsby-plugin-google-analytics/src/gatsby-ssr.js

```diff
@@ -18,91 +18,105 @@ exports.onRenderBody = (
   { setHeadComponents, setPostBodyComponents },
   pluginOptions
 ) => {
-  if (process.env.NODE_ENV === `production`) {
-    let excludeGAPaths = []
-    if (typeof pluginOptions.exclude !== `undefined`) {
-      const Minimatch = require(`minimatch`).Minimatch
-      pluginOptions.exclude.map(exclude => {
-        const mm = new Minimatch(exclude)
-        excludeGAPaths.push(mm.makeRe())
-      })
-    }
+  if (process.env.NODE_ENV !== `production`) {
+    return null
+  }
+
+  // Lighthouse recommends pre-connecting to google analytics
+  setHeadComponents([
+    <link
+        rel="preconnect"
```

Comment:
```suggestion
        rel="dns-prefetch preconnect"
```

Reply:
Is it supported to specify multiple resource hints?

Reply:
yeah it's actually best practice to have it this way so the browser will use preconnect if it supports it (all modern browsers) but will fallback to dns-prefetch if it doesnt (ie11)

Reply:
Couldn't find any documentation on that, but I trust you 😜